infra/sentry/tasks/certificate.yml

```diff
@@ -0,0 +1,47 @@
+- name: Pip uninstall dependencies
+  pip:
+    name: ['requests', 'six', 'urllib3']
+    state: absent
+
+- name: Yum reinstall dependecies
+  yum:
+    name: ['python-requests', 'python-six', 'python-urllib3']
```

Comment:
Je ne comprend pas pourquoi désinstaller et ré-installer ?
Peut-on faire un re-install directement ? ou install if not exists ?

Reply:
Il y a des problèmes de compatibilité (notamment urllib3), j'ai été obligé de faire ça pour que le processus fonctionne dans 100% des cas.